addons/account/models/account_journal.py

```diff
@@ -34,7 +34,7 @@ class AccountJournal(models.Model):
     _description = "Journal"
     _order = 'sequence, type, code'
     _inherit = ['portal.mixin',
-                'mail.alias.mixin',
+                'mail.alias.mixin.light',
```

Comment:
I'm surprised applying the light mixin does not change tests, aren't we supposed to test that we don't create aliases anymore for journals that don't need one? (aka not purchase / sale etc).